### PR TITLE
Asset automation

### PR DESCRIPTION
**Pull request by jspaleta**
Add after_deploy travis automation to generate sha512 checksum file needed for Sensu asset config.

After_deploy actions:
1. generate sha512 checksum file based on existence of sha256 file 
2. upload sha512 checksum file into tagged release

Small adjustment to name of checksum file generated by goreleaser to indicate its sha256 checksums.

Closes https://github.com/sensu/sensu-slack-handler/issues/4

### CODE REVIEW

File: travis/generate-sha512sum.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+
+if [ -d dist ]; then
+  files=( dist/*sha256-checksums.txt )  
+  file=`basename ${files[0]}`
```
Comment (nikkictl) [inline travis/generate-sha512sum.sh]: https://github.com/koalaman/shellcheck/wiki/SC2006

File: travis/github-release-upload.sh [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+# Get ID of the asset based on given filename.
+eval $(echo "$response" | grep -m 1 "id.:" | grep -w id | tr : = | tr -cd '[[:alnum:]]=')
+echo $id
+[ "$id" ] || { echo "Error: Failed to get release id for tag: $tag"; echo "$response" | awk 'length($0)<100' >&2; exit 1; }
+
+# Upload asset
+echo "Uploading asset... "
+
+# Construct url
+GH_ASSET="https://uploads.github.com/repos/$repo_slug/releases/$id/assets?name=$(basename $filename)"
```
Comment (nikkictl) [inline travis/github-release-upload.sh]: https://github.com/koalaman/shellcheck/wiki/SC2154

**Review by nikkictl — COMMENTED** [review]
I learned about https://www.shellcheck.net from @ccressent, it's super useful!

**Comment by jspaleta** [comment]
retagging a release in my fork to ensure updates to shell scripts are still operational

**Comment by jspaleta** [comment]
cool.  I was able to generate a new release in my fork using the cleaned up shell scripts.

**Pull request merged**